recipes/cloudspades/meta.yaml

```diff
@@ -50,7 +50,5 @@ about:
   doc_url: "https://github.com/ablab/spades/tree/cloudspades-ismb"
 
 extra:
-  additional-platforms:
-    - linux-aarch64
```

Comment:
What was the problem with aarch64 here ?

Reply:
It fails to compile on ARM

Reply:
I guess it failed to compile due to the usage of signed chars.
The support has been removed with this PR - https://github.com/bioconda/bioconda-recipes/pull/51456/files#diff-be671f9588fb1b742546ddcfd20773418719432b8e42799df8df52a9e7a3df9eL8 (`-fsigned-char` is gone)

What is the correct way to pass CXXFLAGS for this recipe ?

Reply:
It seems to work correctly without any flags but the previous commits that borrowed the CXX flags from the regular `spades` recipe built, however the resulting installation had errors when running (same for the current `spades` recipe, which remains unresolved). AFAIK, there isn't an example of a proper spades ARM build on bioconda and the existing `spades` (regular) build is currently bugged and unresolved.

Reply:
I know just about nothing about compilation, so I'm able to identify that there is an issue, but powerless to implement meaningful fixes.